library/general/src/lib/installation/proposal_client.rb

```diff
@@ -132,6 +137,11 @@ def run
     #     The caller will format each item as a HTML list item (`<li>`). The
     #     proposal can contain hyperlinks with IDs listed in the list `links`.
     #
+    #   * **`"label_proposal"`** [Array<String>, nil]
```

Comment:
I'd would be probably against reusing the `raw_proposal` key. Defining yet another key does not look nice but's the only way how to know that the client really handles the `cramped_label_mode` correctly.

Think about some products handled outside the YaST team (e.g. OES, SAP...), maybe they would like to use this all-in-one dialog too. But they would like to use different proposal clients, which might not have support for this new mode...

They might try and see that it works (in almost all cases), but they miss a special error/corner case when the proposal suddenly returns rich text instead of plain text label and then it breaks...

Reusing the same key with a different meaning/semantic does not look right to me... When using a new key we can easily check the result and if the key is missing then display an error saying that the module does not support the simplified proposal mode .